### ISSUE TITLE
Transaction UI 2.0 (part 2)

### ISSUE THREAD

**Issue by bgptr** [raedahgroup/dcrios#578]
Remaining tasks after #569

- failed status indication of the transaction on `history` and `txdetails` views
- green backgrounded additional info label value of vote transaction on `history` view
- introducing new ticket types (`revoked`, `expired`, `missed`) on `history` and `txdetails` views
- wallet select control on `history` view
- `From` and `To account` rows on the detail view of a received transaction 
- `From account` and `To` rows on the detail view of a sent transaction

**Comment by bgptr** [comment]
This issue was split into multiple issues: #584, #585,#586, #587, #588
